datasource/hkobserverfactory.js

```diff
@@ -18,6 +19,26 @@ for (let key in classes)
 	clients[C.getType()] = C;
```

Comment:
ObserverClient and ConfigurableObserverClient share the same type. How to reliably know which one will be instantiate? What is the use case behind allowing "default" clients? In my mind, they seem more like abstract classes.

Reply:
I've added a getType to the configurable observer client

Reply:
> What is the use case behind allowing "default" clients? In my mind, they seem more like abstract classes.

Reply:
https://github.com/ibm-hyperknowledge/hkpy/pull/26#discussion_r762149593